src/publish.proj

```diff
@@ -29,18 +29,26 @@
   <PropertyGroup>
     <PackageDownloadDirectory Condition="'$(DownloadDirectory)' == ''">$(PackagesDir)AzureTransfer\$(ConfigurationGroup)</PackageDownloadDirectory>
     <FinalPublishPattern>$(PackageDownloadDirectory)\**\*.nupkg</FinalPublishPattern>
+    <FinalPublishPrivatePattern>$(PackageDownloadDirectory)\**\*Private*.nupkg</FinalPublishPrivatePattern>
```

Comment:
I'm OK with this for now but I suspect that we will end up with some mechanism similar to our signing where someone can opt-out of shipping in their project file and we will find a marker file or something so we know to opt them out.

Reply:
Agreed, that sounds much better than patterns.